<launch>
	<node name="patrol"
		pkg="whs_navigation"
		type="smach_patrol_main.py"
		args="$(find whs_navigation)/poses/b5203.txt"
		output="screen">
		<remap from="laptop_charge" to="laptop_charge"/>
		<remap from="power_system_event" to="mobile_base/events/power_system"/>
		<remap from="sound_command" to="mobile_base/commands/sound"/>
	</node>
</launch>
